src/main/kotlin/no/nav/syfo/infrastructure/clients/pdfgen/PdfGenClient.kt

```diff
@@ -77,7 +77,7 @@ class PdfGenClient(
     companion object {
         private const val API_BASE_PATH = "/api/v1/genpdf/isarbeidsuforhet"
         const val FORHANDSVARSEL_PATH = "/forhandsvarsel-om-avslag-pa-sykepenger"
-        const val OPPFYLT_PATH = "/vurdering-av-arbeidsuforhet"
+        const val VURDERING_PATH = "/vurdering-av-arbeidsuforhet"
```

Comment:
Det blir altså en felles url for vurdering-pdfene, og så må frontend bestemme innhold og struktur på OPPFYLT og AVSLAG utover dette.